packages/protocol/contracts/governance/Governance.sol

```diff
@@ -325,13 +323,51 @@ contract Governance is
     emit ExecutionStageDurationSet(executionStageDuration);
   }
 
+  /**
+   * @notice Updates the floor of the participation baseline.
+   * @param _participationFloor The value at which the baseline is floored.
+   */
+  function setParticipationFloor(int256 _participationFloor) external onlyOwner {
+    require(_participationFloor != participationFloor && isFraction(_participationFloor));
+    participationFloor = _participationFloor;
+    emit ParticipationFloorSet(_participationFloor);
+  }
+
+  /**
+   * @notice Updates the weight of the new participation in the baseline update rule.
+   * @param _participationUpdateCoefficient The weight of the new participation.
+   */
+  function setParticipationUpdateCoefficient(
+    int256 _participationUpdateCoefficient
+  )
+    external
+    onlyOwner
+  {
+    require(
+      _participationUpdateCoefficient != participationUpdateCoefficient &&
+      isFraction(_participationUpdateCoefficient)
+    );
+    participationUpdateCoefficient = _participationUpdateCoefficient;
+    emit ParticipationUpdateCoefficientSet(_participationUpdateCoefficient);
+  }
+
+  /**
+   * @notice Updates the proportion to compute the critical baseline.
+   * @param _criticalBaselineLevel The weight of the new participation.
+   */
+  function setCriticalBaselineLevel(int256 _criticalBaselineLevel) external onlyOwner {
+    require(_criticalBaselineLevel != criticalBaselineLevel && isFraction(_criticalBaselineLevel));
+    criticalBaselineLevel = _criticalBaselineLevel;
+    emit CriticalBaselineLevelSet(_criticalBaselineLevel);
+  }
+
   /**
    * @notice Updates the ratio of yes:yes+no votes needed for a specific class of proposals to pass.
    * @param destination The destination of proposals for which this threshold should apply.
    * @param functionId The function ID of proposals for which this threshold should apply. Zero
    *   will set the default.
    * @param threshold The threshold.
-   * @dev If no constitution is explicitly set the default is a simple majority, i.e. 1:2.
+   * @dev If no constitution is explicitly set the default is a 70% supermajority.
```

Comment:
Jared and I shared concerns that defaulting to 50% would allow potentially sensitive governables to be changed before they were added to the constitution. At this level, passing a proposal not mentioned in the constitution would require enough support to lower the threshold. Definitely open to discussion on whether this is necessary in practice